src/keychain.ts

```diff
@@ -1,13 +1,51 @@
-import { getPassword, setPassword } from 'keytar';
+import { getPassword } from 'keytar';
+import { getConfig } from './config-new';
 
 const SERVICE = 'hutte-io';
 
+const REMOVAL_INSTRUCTIONS = `
+You can now remove the 'hutte-io' entry from the keychain of your OS.
+
+MacOS
+
+- Open 'Keyhain Access.app'
+- Search for 'hutte-io' of kind application password
+- Right click and delete
+
+Linux
+
+- TBD
+
+Windows
+
+- TBD
+`;
+
 async function getUserApiToken(params: { userId: string }): Promise<string | null> {
-  return await getPassword(SERVICE, params.userId);
+  const config = await getConfig();
+  const token = config.get('token', true);
+  if (token) {
+    return token;
+  }
+  // for backwards compatibility with keytar
+  try {
```

Comment:
While I like the backwards compatibility, I hesitate if it's worth to maintain this complexity in the long run. Taking into account that the new version of the CLI also has a breaking change with the `sf` migration, don't you think we should not make this backwards compatible and force users to re-authenticate with the new system?

Reply:
dependency and backwards compatibility now removed